File: collection/map_and_set/iter/iter.go
package main

import "fmt"

func main() {
	fmt.Println("iterate map")
	m := map[string]int{
		"one":   1,
		"two":   2,
		"three": 3,
	}

	for k, v := range m {
		fmt.Println(k, v)
	}

	fmt.Println("golang does not have Set in its standard library. Use map with a constant value to create a set")
	const t = true
	set := map[string]bool{"one": t, "two": t, "three": t}
	fmt.Println(set)
}

File: collection/map_and_set/iter/iter.py

d = {"a": 1, "b": 2, "c": 3}

print("with keys")

for k in d:
    print(k)

for k in d.keys():
    print(k)

print("with values")
for k in d.values():
    print(k)

print("with keys and values")
for k, v in d.items():
    print(k, v)

print("iterate set")
set = {1,2,3}
for e in set:
    print(e)
